larray_editor/utils.py

```diff
@@ -595,3 +595,51 @@ def get_sample(data, maxsize):
 def get_sample_indices(data, maxsize):
     flat_indices = np.arange(0, data.size, get_sample_step(data, maxsize))
     return np.unravel_index(flat_indices, data.shape)
+
+
+class RecentFileList(object):
+    MAX_RECENT_FILES = 10
+
+    def __init__(self, list_name, actions=False, parent=None):
```

Comment:
Unsure having an explict actions argument is useful. You could rename parent to action_parent and use `if action_parent is not None` instead.